parachain/runtime/local/src/lib.rs

```diff
@@ -169,11 +169,11 @@ const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
 parameter_types! {
 	pub const Version: RuntimeVersion = VERSION;
 	pub const BlockHashCount: BlockNumber = 2400;
-	/// We allow for 2 seconds of compute with a 6 second average block time.
+	/// We allow for 0.5 seconds of compute with a 12 second average block time.
```

Comment:
hm mind explaining what this is for or linking to a relevant substrate doc where i can read about it?

Reply:
Yeah so while going some rabbit-holes while testing header verification (that's a story for another time), I discovered that the previous settings were incorrect and could lead to relay chain validators failing to validate our blocks. Basically the maximum block weight of a parachain block is bounded by the maximum weight of relay chain block (0.5s of weight).

I copied this over from statemint/rococo parachains, as its a more realistic figure: https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/statemint-common/src/lib.rs#L86

Strictly speaking this change isn't relevant to Ropsten or the London hardfork. Only made it while troubleshooting an issue.